client/src/App.js

```diff
@@ -1,7 +1,40 @@
 import "./App.css";
+import "./components/Navigation.css";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import Navigation from "./components/Navigation";
+import Friends from "./components/Friends";
+import Map from "./components/Map";
+import Profile from "./components/Profile";
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <Router>
+      <div className="grid-layout-app">
+        <header className="header">
+          <h1>Header</h1>
+        </header>
+        <main className="main">
+          <Switch>
+            <Route path="/friends">
+              <Friends />
+            </Route>
+            <Route path="/map">
+              <Map />
+            </Route>
+            <Route exact path="/">
+              <Profile />
+            </Route>
+            <Route path="*">
+              <h2>No match! Go Back 🤓 </h2>
+            </Route>
+          </Switch>
```

Comment:
I don't see why `Friends`, `Map` or `Profile` should be  part of this PR. This is about the Navigation and not placeholder components. It would be cleaner if you'd just display different headlines depending on the url visited